### ISSUE TITLE
bug: 'NoneType' object is not subscriptable when using Multipart

### ISSUE THREAD

**Issue by hoangphucITJP** [bentoml/BentoML#3252]
### Describe the bug

```
>>> client.combine(a='a', b='b')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/X/.pyenv/versions/X/lib/python3.9/site-packages/bentoml/client.py", line 57, in _sync_call
    return asyncio.run(self._call(inp, _bentoml_api=_bentoml_api))
  File "/home/X/.pyenv/versions/3.9.11/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/home/X/.pyenv/versions/3.9.11/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/X/.pyenv/versions/X/lib/python3.9/site-packages/bentoml/client.py", line 126, in _call
    fake_resp = await api.input.to_http_response(inp, None)
  File "/home/X/.pyenv/versions/X/lib/python3.9/site-packages/bentoml/_internal/io_descriptors/multipart.py", line 276, in to_http_response
    *tuple(
  File "/home/X/.pyenv/versions/X/lib/python3.9/site-packages/bentoml/_internal/io_descriptors/multipart.py", line 277, in <genexpr>
    io_.to_http_response(obj[key], ctx) for key, io_ in self._inputs.items()
TypeError: 'NoneType' object is not subscriptable
```

### To reproduce

Provide simple multipart service
```
import bentoml
from bentoml.io import Multipart, Text

svc = bentoml.Service('test')


@svc.api(input=Multipart(a=Text(), b=Text()), output=Text())
def combine(a, b):
    return ''

```

### Environment

bentoml: 1.0.10
python: 3.9.11
OS: Ubuntu 18.04

**Comment by hoangphucITJP** [comment]
Look like the problem is in this method
https://github.com/bentoml/BentoML/blob/main/src/bentoml/client.py#L54

**Comment by sauyon** [comment]
Hiya! I've actually solved this, it's a pretty silly bug for multipart.

**Comment by sauyon** [comment]
Should be fixed by #3253.

**Comment by hoangphucITJP** [comment]
It works now. Thanks for support
